packages/flutter/lib/src/cupertino/text_selection_toolbar.dart

```diff
@@ -920,6 +955,18 @@ class _RenderCupertinoTextSelectionToolbarItems extends RenderBox with Container
       if (childParentData.shouldPaint) {
         final Offset childOffset = childParentData.offset + offset;
         context.paintChild(child, childOffset);
+
+        // backButton is not in the children list, that's why its
+        // childParentData.nextSibling is null. So either when there's a
```

Comment:
"backButton is a slotted child and is not in the children list, so its childParentData.nextSibling is null. ..."